gno.land/cmd/gnoland/main.go

```diff
@@ -105,6 +112,20 @@ func (c *gnolandCfg) RegisterFlags(fs *flag.FlagSet) {
 		"localhost:26657",
 		"replacement for '%%REMOTE%%' in genesis",
 	)
+
+	fs.StringVar(
+		&c.txIndexerType,
+		"tx-indexer-type",
+		null.IndexerType,
+		fmt.Sprintf("type of transaction indexer [%s, %s]", null.IndexerType, file.IndexerType),
+	)
```

Comment:
nit: have a public list of all available indexers to avoid missing to change this description in the future.

Reply:
I've made it easier to render this list:
[c09a831](https://github.com/gnolang/gno/pull/546/commits/c09a8317a7e28604298fff2163e987b53f3d8955)

Not sure I want to define it as a public list, as the base indexer service should not know about the underlying implementations (and thus their types) - it would require me to slightly tweak how the indexer service and its underlying implementations interact